oraclelinux/content.md

```diff
@@ -39,5 +39,4 @@ Please review the [Oracle Container Runtime for Docker User's Guide](https://doc
 
 For Oracle Linux users without a paid support subscription, the following resources are available:
 
--	The [Oracle Linux Forum](https://community.oracle.com/community/server_%26_storage_systems/linux/oracle_linux) on the [Oracle Technology Network Community](https://community.oracle.com/welcome).
--	The `#oracle-linux` IRC channel on Freenode.
+- The [Oracle Linux Space](https://community.oracle.com/community/groundbreakers/server_&_storage_systems/linux/oracle_linux) on the [Oracle Technology Network Community](https://community.oracle.com/welcome).
```

Comment:
Travis is going to come back telling you to switch the space after your bullet point here into a tab, btw (`-<tab>The ...`).

Reply:
Heh, and `\ No newline at end of file`. :+1:

Reply:
Ha, and I removed the tab because I didn't like it. 😂 Guess I'll have to put it back.